dashboard/config/locales/en.yml

```diff
@@ -696,20 +692,6 @@ en:
       body: "Students can explore concepts from our Computer Science Principles course hands-on using these digital tools. Use the widgets on their own, or create a single-concept lesson for your class."
       audience: "Ages 14+"
       cta: "Learn how"
-  gallery:
```

Comment:
Deleting these gallery strings is the relevant change here.  Everything else is just an auto change of 'single-quotes' to "double-quotes"